Misc/NEWS.d/next/Core and Builtins/2019-08-13-18-05-20.bpo-37840.elLCci.rst

```diff
@@ -0,0 +1,2 @@
+Fix handling of negative indices in :c:member:`~PySequenceMethods.sq_item`
```

Comment:
Should this be PySequenceMethods.sq_item or PySequence_GetItem?

Reply:
PySequenceMethods.sq_item